Makefile.js

```diff
@@ -84,12 +84,7 @@ const NODE = "node ", // intentional extra space
  * @private
  */
 function getTestFilePatterns() {
-    return ls("tests/lib/").filter(pathToCheck => test("-d", `tests/lib/${pathToCheck}`)).reduce((initialValue, currentValues) => {
-        if (currentValues !== "rules") {
-            initialValue.push(`"tests/lib/${currentValues}/**/*.js"`);
-        }
-        return initialValue;
-    }, ["\"tests/lib/rules/**/*.js\"", "\"tests/lib/*.js\"", "\"tests/bin/**/*.js\"", "\"tests/tools/**/*.js\""]).join(" ");
+    return ["\"tests/lib/**/*.js\"", "\"tests/bin/**/*.js\"", "\"tests/tools/**/*.js\""].join(" ");
```

Comment:
I think this is same.

- `tests/lib/*.js` + `tests/lib/rules/**/*.js` + `tests/lib/${x}/**/*.js` (x is the directories on `tests/lib` except `tests/lib/rules`) = `tests/lib/**/*.js`.

Reply:
I assume `"tests/lib/*.js"` is covered by the new pattern as well?

Reply:
Yes!